projects/angular-showcase/src/app/examples/header-showcase/header-showcase.component.html

```diff
@@ -4,10 +4,12 @@
   [environment]="'dev'"
   [environmentColor]="'darkgreen'"
 >
-  <a routerLink="/">Home</a>
-  <a routerLink="/navigation1">Another module</a>
-  <button type="button" [sbbDropdown]="dropdown">Navigation 1</button>
-  <button type="button">Navigation 2</button>
+  <a sbbNavbutton routerLink="/">Home</a>
```

Comment:
Is it necessary to use the navbutton component for a tags? If possible, I would like to only have to use it for buttons.

Reply:
No, I thought you meant to have it for all of them, but it's a minor change to have it for buttons only.